docs/recipes/vue.md

```diff
@@ -36,9 +38,27 @@ const Vue = require('vue');
 // Setup Vue.js to remove production tip
 Vue.config.productionTip = false;
 
+// Setup Vue files to be processed by `require-extension-hooks-vue`
+hooks('vue').plugin('vue').push();
+// Setup Vue and JS files to be processed by `require-extension-hooks-babel`
+hooks(['vue', 'js']).plugin('babel').push();
+```
+
+When you are using ES6 modules, use the dist-file of Vue. Not doing that may cause a not completely accessible (and thus not testable) Vue model.
+
+```js
+import browserEnv from 'browser-env';
+import hooks from 'require-extension-hooks';
+import Vue from 'vue/dist/vue';
```

Comment:
Does this type of import not require webpack? I didn't think this worked out of the box with babel, etc. yet.

Reply:
Not sure of the point of this example, same as above but es imports. 

@sindresorhus can you confirm @OmgImAlexis is also correct.